string name
string acceptor
---
bool removed
